### DISCUSSION TITLE
Some transactions not showing in recent activity in the wallet

### DISCUSSION THREAD

**Discussion by kayamallow** [phantom/docs#160] [General]
There is a display issue with some SOL transfer transactions that's causing us some hassle. The issue happens when we refund users that lose auctions. The transaction does not display in their recent activity so they never see the refund.

Example transactions that have had the issue:
https://explorer.solana.com/tx/SfajLM8utXLdSntmbSkAH73WtZzGAoXUXKuatyuyur53dRYaF15PsPxjUqZXis9LKk467hBVm5jARqqQFr2kEN5
https://explorer.solana.com/tx/4BwMr4Nh39S5w2wCrPBu2fngfJZ2hS45ZmiFaXmbsjmo4RZNS2jHkutSaWmK6UsMbXRp2VCEX8eBZGgUBXBAN1GA

If you need more I can send. Thanks.

**Comment by adamdelphantom** [comment]
Hi @kayamallow,

Thank you for writing in! We are investigating this internally. I will let you know about any updates here.

**Comment by kayamallow** [comment]
Any updates yet @adamdelphantom?

This is something we deal with a lot and obviously isn't a reputation we want to build when we are refunding all users correctly.

![image](https://github.com/phantom/docs/assets/141120062/40e8b9a8-9d56-4294-92a5-ab4d8b0fac5f)

**Comment by meditatingsloth** [comment]
@adamdelphantom It appears the issue happens when the user receives funds from a PDA using lamport changes instead of a transfer ix
